src/theme/variables.css

```diff
@@ -1,6 +1,14 @@
 /* Ionic Variables and Theming. For more info, please see:
 http://ionicframework.com/docs/theming/ */
 
+/** Font faces **/
+@font-face {
+  font-family: 'Oswald';
```

Comment:
Can we add a fallback here? If you don't know how, look at this article: https://www.w3schools.com/css/css_font.asp

Reply:
Jk, I misunderstood.